package.json

```
@@ -114,11 +109,11 @@
     "cypress": "13.14.1",
     "eslint": "9.9.1",
     "eslint-config-prettier": "9.1.0",
-    "eslint-define-config": "2.1.0",
     "eslint-plugin-jsdoc": "50.2.2",
     "eslint-plugin-prettier": "5.2.1",
     "eslint-plugin-unicorn": "55.0.0",
     "eslint-plugin-vitest": "0.5.4",
+    "knip": "^5.30.1",
```

We use pinned versions only.

```suggestion
    "knip": "5.30.1",
```

---

Good one! Changing!

---

Solved ✅